gobblin-service/src/test/java/org/apache/gobblin/service/modules/orchestration/UserQuotaManagerTest.java

```diff
@@ -68,4 +67,17 @@ public void testExceedsQuotaThrowsException() throws Exception {
       this._quotaManager.checkQuota(dags.get(1).getNodes().get(0), false);
     });
   }
+
+  @Test
+  public void testMultipleRemoveQuotasNegative() throws Exception {
```

Comment:
technically aren't you testing that multiple (specifically, repeated) removes *don't* leave the quota negative.  method name suggests that you're testing that they do go negative.

minor: you aren't truly verifying whether a value is pos/neg, but just that repeated removes are ignored.  that may be fine, but if it's really what you need to ensure, it might be best to look at it specifically.

Reply:
Renamed this to testMultipleRemoveQuotasIdempotent to better reflect what I'm testing for